src/systems/ackermann_steering/AckermannSteering.cc

```diff
@@ -216,48 +234,62 @@ void AckermannSteering::Configure(const Entity &_entity,
   if (!links.empty())
     this->dataPtr->canonicalLink = Link(links[0]);
 
-  // Ugly, but needed because the sdf::Element::GetElement is not a const
-  // function and _sdf is a const shared pointer to a const sdf::Element.
-  auto ptr = const_cast<sdf::Element *>(_sdf.get());
+  if (_sdf->HasElement("steering_only"))
+  {
+    this->dataPtr->steeringOnly = _sdf->Get<bool>("steering_only");
+    gzwarn << "Using steering only mode: " << this->dataPtr->steeringOnly
```

Comment:
I think that info or debug is suitable here as it isn't a hazardous situation, but rather something that users have opted into?

Reply:
Good point, will set to msg.